frontend/src/components/TopNavBarRevamp.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { ReactElement } from 'react'
+import { useHistory } from 'react-router-dom'
+import { NavBar, Icon } from 'antd-mobile'
+import styled from 'styled-components'
+
+import 'antd-mobile/dist/antd-mobile.css'
+
+const TitleText = styled.text`
+  font-family: lato;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  line-height: 20px;
+  color: #000000;
+  align-self: center;
+  white-space: nowrap;
+  padding-left: 50px;
+`
+const StyledNavBar = styled(NavBar)`
+  &.am-navbar {
+    height: 70px;
+    background-color: #ffffff; !important
+  }
+  padding: 15px;
+  max-width:100%;
+  position: sticky;
+  top:0;
+  z-index:200;
+  display: flex;
+  flex-direction: row;
+  justify-content: right;
```

Comment:
fix the spacing between : and text!

Reply:
Resolved in commit 8d8bcd18261ca08a4f4b82800eadadc527fb43b1.

Reply:
Resolved in commit 28d833d7fb2a32c68adcdc36fd7f772f555503fe